CMakeLists.txt

```diff
@@ -41,6 +41,16 @@ if(POLICY CMP0045)
       cmake_policy(SET CMP0045 OLD)
 endif(POLICY CMP0045)
 
+# RPATH settings on macOS do not affect install_name
+if(POLICY CMP0068)
```

Comment:
Can we keep them in numerical order?

Reply:
Sure. I didn't notice that was the case already - sorry!